desktop/lib/metaphysics.coffee

```diff
@@ -28,11 +29,11 @@ metaphysics = ({ method, query, variables, req } = {}) ->
       if err?
         return reject err
 
+      # Allow errors in resolving GraphQL data to not affect resolving the whole promise.
+      # However, notify NewRelic.
       if response.body.errors?
         error = new Error JSON.stringify response.body.errors
-        error.status = 404 if some(response.body.errors, ({ message }) -> message.match /Not Found/)
-        error.data = response.body.data
-        return reject error
+        newrelic.noticeError(error, { crash: true })
```

Comment:
Idk if we want to format/group these better. In testing locally, all of these errors will be consolidated and grouped based on the route they originate from, so I think this might be sufficient.

ie- any errors in resolving GraphQL data for the artwork page will all be wrapped into the `GET /artwork/:id` error (w/ the full stack trace from GraphQL errors).